### DISCUSSION TITLE
버전 업데이트 오류

### DISCUSSION THREAD

**Discussion by Chelsea9900** [naverz/zepeto-studio-kor#142] [5. Q&A]
월드 버전을 업데이트하려고 하는데 이 상태에서 1시간이 넘어도 업로드가 되지 않습니다..
원인을 알 수 있을까요?

<img width="768" alt="스크린샷 2021-12-24 오후 10 16 07" src="https://user-images.githubusercontent.com/93331248/147355396-c7fea0d7-86f0-4c02-87fe-f18b475e853c.png">
.

**Comment by Chelsea9900** [comment]
해결됐습니다! 4시간 정도 지나고 나서 업로드 실패라고 떴습니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
어제 오늘 극심한 트래픽을 서버가 감당하지 못해서 월드 서비스가 많이 불안정했었습니다.
현재 서버를 대량 증설하여 문제를 해결해뒀는데요.
이제 다시 재등록 해주시면 이제 정상적으로 업로드가 될 것으로 생각됩니다.
불편을 끼쳐드려 정말 죄송합니다.
